lib/mmbooking_Clone_web/components/layouts/root.html.heex

```diff
@@ -39,20 +39,66 @@
           <.link
             href={~p"/"}
             class="text-[0.8125rem] leading-6 text-zinc-900 font-semibold hover:text-zinc-700"
-          >
-            Register
-          </.link>
-        </li>
-        <li>
-          <.link
-            href={~p"/log_in"}
-            class="text-[0.8125rem] leading-6 text-zinc-900 font-semibold hover:text-zinc-700"
           >
             Log in
           </.link>
         </li>
       <% end %>
     </ul>
-    <%= @inner_content %>
+    <%= if @current_user do %>
```

Comment:
Why sidebar coming in root.html. What about admin layout?

Reply:
What do you mean by admin layout? Admin can only see the sidebar. normal users cannot see this sidebar. See the below picture. 
[
![Uploading Screenshot from 2024-03-07 11-36-47.png…]()
](url)

Reply:
Don't show menus in welcome page, visitors form page and Visitior home page. The menus only visible for admins not for visitors.

Reply:
@sakthibalan15 Do i need to change anything

Reply:
I have used root.html just like how we used to write code for navbars.

Reply:
@sakthibalan15 told me to search about this.